pkg/volume/csi/csi_mounter.go

```diff
@@ -314,35 +314,24 @@ func (c *csiMountMgr) TearDownAt(dir string) error {
 // 1) if fstype is "", then skip fsgroup (could be indication of non-block filesystem)
 // 2) if fstype is provided and pv.AccessMode == ReadWriteOnly and !c.spec.ReadOnly then apply fsgroup
 func (c *csiMountMgr) applyFSGroup(fsType string, fsGroup *int64) error {
+	klog.Infof("Applying fsGroup %v to %v for volume %v ...", fsGroup, fsType, c.volumeID)
```

Comment:
where insert values can be long I prefer %q.

Reply:
ack, but either way fsgroup's should be small not sure why its a 64 bit int